src/styles/sidebar/components/thread.scss

```diff
@@ -0,0 +1,50 @@
+@use "../../variables" as var;
+
+.thread {
```

Comment:
This CSS is fairly prototype-quality right now—don't worry about the specifics yet, I'll clean it up.